app/jobs/execution_job.rb

```diff
@@ -2,8 +2,10 @@ class ExecutionJob < ApplicationJob
   queue_as :default
 
   def perform execution_id
-    execution = Execution.find execution_id
+    status = system(
+      "rails runner -e #{Rails.env} 'Execution.find(#{execution_id}).run_with_profiler'"
+    )
 
-    execution.run
+    raise Exception, 'can not run_with_profiler' unless status
```

Comment:
Lo dejaría como:

```suggestion
    raise 'Cannot execute run_with_profiler' unless status
```

Así es _runtime_ como creo que debería =P